mentions-generator/src/main/java/de/brandwatch/minianalytics/mentiongenerator/kafka/Consumer.java

```diff
@@ -1,16 +1,114 @@
 package de.brandwatch.minianalytics.mentiongenerator.kafka;
 
+import de.brandwatch.minianalytics.mentiongenerator.model.Mention;
+import de.brandwatch.minianalytics.mentiongenerator.model.Ressource;
+import de.brandwatch.minianalytics.mentiongenerator.postgres.model.Query;
+import de.brandwatch.minianalytics.mentiongenerator.postgres.repository.QueryRepository;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.Field;
+import org.apache.lucene.document.TextField;
+import org.apache.lucene.index.DirectoryReader;
+import org.apache.lucene.index.IndexReader;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.queryparser.classic.ParseException;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.RAMDirectory;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.kafka.annotation.KafkaListener;
 
+import java.io.IOException;
+import java.util.List;
+
 public class Consumer {
 
     private static final Logger logger = LoggerFactory.getLogger(Consumer.class);
 
+    @Autowired
+    private QueryRepository queryRepository;
+
+    @Autowired
+    private Producer producer;
+
+    private Directory memoryIndex = new RAMDirectory();
+    private StandardAnalyzer analyzer = new StandardAnalyzer();
+
+    private IndexWriterConfig indexWriterConfig = new IndexWriterConfig(analyzer);
+    private IndexWriter writer = new IndexWriter(memoryIndex, indexWriterConfig);
+
+    private IndexSearcher searcher;
+    private IndexReader reader;
+
+    private QueryParser queryParser = new QueryParser("text", analyzer);
+
+    public Consumer() throws IOException {
+    }
+
     @KafkaListener(topics = "twitter")
-    public void receive(String message){
-        logger.info("received message='{}'", message);
+    public void receive(Ressource ressource) throws ParseException, IOException {
```

Comment:
Wasn't this renamed at some point already?

```suggestion
    public void receive(Resource resource) throws ParseException, IOException {
```

Reply:
It is possible, that it was renamed on a different branch or in a different module.